packages/devtools_app/lib/src/framework/framework_core.dart

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
 
+/// @docImport '../../devtools_app.dart';
```

Comment:
do we want to import devtools_app.dart here or the specific file that is being referenced? devtools_app.dart just exports a bunch of devtools files.

Reply:
Good catch. For some reason, analysis server is only suggesting `devtools_app.dart` 🫤  I've manually corrected it.